build/configs/cy4390x/st_things/defconfig

```diff
@@ -0,0 +1,1202 @@
+CONFIG_WIFIMGR_SOFTAP_IFNAME="wl2"
+CONFIG_WIFIMGR_STA_IFNAME="wl1"
+
+#
+# Network utilities
+#
+CONFIG_NETUTILS_NETLIB=y
+
+#
+# Audio Support
+#
+CONFIG_AUDIO=y
+# CONFIG_AUDIO_MULTI_SESSION is not set
+
+#
+# Audio Buffer Configuration
+#
+# CONFIG_AUDIO_LARGE_BUFFERS is not set
+CONFIG_AUDIO_NUM_BUFFERS=2
+CONFIG_AUDIO_BUFSIZE=2048
+# CONFIG_AUDIO_DRIVER_SPECIFIC_BUFFERS is not set
+CONFIG_AUDIO_RESAMPLER_BUFSIZE=4096
+CONFIG_AUDIO_DRIVER_SPECIFIC_BUFFER_MGMT=y
+
+#
+# Exclude Specific Audio Features
+#
+# CONFIG_AUDIO_EXCLUDE_VOLUME is not set
+# CONFIG_AUDIO_EXCLUDE_BALANCE is not set
+CONFIG_AUDIO_EXCLUDE_EQUALIZER=y
+# CONFIG_AUDIO_EXCLUDE_TONE is not set
+CONFIG_AUDIO_EXCLUDE_PAUSE_RESUME=y
+CONFIG_AUDIO_EXCLUDE_STOP=y
+CONFIG_AUDIO_EXCLUDE_FFORWARD=y
+CONFIG_AUDIO_EXCLUDE_REWIND=y
+# CONFIG_AUDIO_CUSTOM_DEV_PATH is not set
+
+#
+# Media Support
+#
+# CONFIG_MEDIA is not set
+# CONFIG_AUDIO_CODEC is not set
+
+#
+# File Systems
+#
+# CONFIG_DISABLE_MOUNTPOINT is not set
+# CONFIG_DISABLE_PSEUDOFS_OPERATIONS is not set
+CONFIG_FS_READABLE=y
+CONFIG_FS_WRITABLE=y
+# CONFIG_FS_AIO is not set
+# CONFIG_FS_NAMED_SEMAPHORES is not set
+CONFIG_FS_MQUEUE_MPATH="/var/mqueue"
+CONFIG_FS_SMARTFS=y
+
+#
+# SMARTFS options
+#
+CONFIG_SMARTFS_ERASEDSTATE=0xff
+CONFIG_SMARTFS_MAXNAMLEN=32
+# CONFIG_SMARTFS_MULTI_ROOT_DIRS is not set
+CONFIG_SMARTFS_ALIGNED_ACCESS=y
+# CONFIG_SMARTFS_BAD_SECTOR is not set
+# CONFIG_SMARTFS_DYNAMIC_HEADER is not set
+# CONFIG_SMARTFS_JOURNALING is not set
+# CONFIG_SMARTFS_SECTOR_RECOVERY is not set
+CONFIG_FS_PROCFS=y
+
+#
+# Exclude individual procfs entries
+#
+# CONFIG_FS_PROCFS_EXCLUDE_PROCESS is not set
+# CONFIG_FS_PROCFS_EXCLUDE_UPTIME is not set
+# CONFIG_FS_PROCFS_EXCLUDE_VERSION is not set
+# CONFIG_FS_PROCFS_EXCLUDE_MTD is not set
+# CONFIG_FS_PROCFS_EXCLUDE_PARTITIONS is not set
+# CONFIG_FS_PROCFS_EXCLUDE_SMARTFS is not set
+CONFIG_FS_ROMFS=y
+
+#
+# Block Driver Configurations
+#
+CONFIG_RAMDISK=y
+
+#
+# MTD Configuration
+#
+CONFIG_MTD=y
+CONFIG_MTD_PARTITION=y
+CONFIG_MTD_PARTITION_NAMES=y
+CONFIG_MTD_PROGMEM=y
+CONFIG_MTD_FTL=y
+
+#
+# MTD_FTL Configurations
+#
+CONFIG_MTD_CONFIG=y
+# CONFIG_MTD_BYTE_WRITE is not set
+
+#
+# MTD Device Drivers
+#
+CONFIG_MTD_M25P=y
+
+#
+# SPI-based M25 flash options
+#
+CONFIG_M25P_SPIMODE=0
+CONFIG_M25P_MANUFACTURER=0xEF
+CONFIG_M25P_MEMORY_TYPE=0x40
+CONFIG_M25P_SUBSECTOR_ERASE=y
+# CONFIG_RAMMTD is not set
+CONFIG_MTD_SMART=y
+
+#
+# SMART Device options
+#
+CONFIG_MTD_SMART_SECTOR_SIZE=4096
+# CONFIG_MTD_SMART_WEAR_LEVEL is not set
+# CONFIG_MTD_SMART_ENABLE_CRC is not set
+# CONFIG_MTD_SMART_SECTOR_ERASE_DEBUG is not set
+# CONFIG_MTD_SMART_ALLOC_DEBUG is not set
+
+#
+# System Logging
+#
+# CONFIG_SYSLOG is not set
+# CONFIG_SYSLOG_TIMESTAMP is not set
+
+#
+# Database
+#
+# CONFIG_ARASTORAGE is not set
+
+#
+# Memory Management
+#
+# CONFIG_DISABLE_REALLOC_NEIGHBOR_EXTENSION is not set
+# CONFIG_MM_SMALL is not set
+CONFIG_MM_REGIONS=1
```

Comment:
Please check this commit 
(https://github.com/Samsung/TizenRT/commit/a0c022e5f5f021af241b1ce3eb6dc67eeb01afb7)
CONFIG_MM_NHEAPS=1
CONFIG_RAM_REGIONx_START
CONFIG_RAM_REGIONx_SIZE
are needed.